MANIFEST.in

```diff
@@ -5,3 +5,4 @@ include requirements.txt
 include LICENSE
 include README.md
 global-exclude *.py[cod] __pycache__ *~ *.bak
+exclude js/**/*
```

Comment:
I trust you on this more than I trust myself haha.  I tried it and it worked the same, so I'll push that commit and then merge.